vector/src/main/res/values/audio_waveform_attr.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be moved to `https://github.com/vector-im/element-android/tree/main/library/ui-styles/src/main/res/values` and its name should start with `stylable_`

Reply:
Done.